### PR TITLE
Added tests for Microsoft.VisualBasic.CompilerServices.Conversions #1…

### PR DESCRIPTION
**Pull request by OzieGamma**
…4344

Hi,

I am trying to get started with contributing to corefx. Is this the kind of code coverage you are looking for?
 
@dotnet-bot test code coverage please

**NB: I'll squash this branch with more tests if I am going in the correct direction**

### CODE REVIEW

**Comment by OmarTawfik** [comment]
@OzieGamma this is great to see! :)
You might want to take a look at the respective C# tests as well for coverage examples: https://github.com/dotnet/corefx/tree/master/src/Microsoft.CSharp/tests
cc @JonHanna in case you're working on similar items.

**Comment by stephentoub** [comment]
@dotnet-bot test NETFX x86 Release Build please

**Comment by stephentoub** [comment]
@dotnet-bot test this please

File: src/Microsoft.VisualBasic/tests/ConversionsTests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        }
+
+        [Theory]
+        [InlineData("yes")]
+        [InlineData("contoso")]
+        public void ToBoolean_String_ThrowsOnInvalidFormat(string str)
+        {
+            Assert.Throws<InvalidCastException>(() => Conversions.ToBoolean(str));
+        }
+
```
Comment (OmarTawfik) [inline src/Microsoft.VisualBasic/tests/ConversionsTests.cs]: nit: blank line.

**Comment by OmarTawfik** [comment]
Other than the nit, I think this is ready to merge.

**Comment by OzieGamma** [comment]
What does NIT mean?

**Comment by stephentoub** [comment]
nit == minor issue... as in "nitpicking" https://en.m.wiktionary.org/wiki/nitpick

**Comment by JonHanna** [comment]
If you plan to take the Microsoft.CSharp tests as a guide to what needs coverage it'd also be worth looking at some of the dynamic tests within System.Linq.Expressions which is picked up for the coverage of Microsoft.CSharp since the use of `dynamic` in C# needs types from both and so the same tests genuinely test both. There may be tests there for which there are useful VB equivalents. There might be some benefit in looking at System.Dynamic.Runtime's tests too.

**Comment by OzieGamma** [comment]
@JonHanna I am not familiar at all with the code/tests. That's why I am trying to pick up the easy PRs. I'll have a look and do another PR if needed.

**Comment by JonHanna** [comment]
Oh, it would definitely want to be separate to this, probably several PRs. Or maybe there's not much worth taking as a starting point, but as Omar says there's likely stuff there that could help make some such PRs easier to plan.

**Comment by OmarTawfik** [comment]
LGTM. Thanks @OzieGamma 
Let us know if we can provide any other pointers.

**Pull request merged**